package.json

```diff
@@ -8,6 +8,9 @@
     "@ethersproject/providers": "^5.0.23",
     "@ethersproject/units": "^5.0.10",
     "@headlessui/react": "dev",
+    "@lingui/core": "^3.13.3",
```

Comment:
i haven't used that lib before. i encountered with react-i18next and react-intl, i guess they are more widely used. do you think `lingui` better fits our needs?

Reply:
Same here.
https://github.com/gmx-io/gmx-interface/pull/130
Please check above pr. I used react-i18next.

Reply:
picked lingui cos i think the syntax is preferable, it is easier to read and edit with the default language, vs with i18n which would require checking two files to figure out the content